### ISSUE TITLE
transforms.Resize Seems to Not Work With 'int'

### ISSUE THREAD

**Issue by Huud** [jbohnslav/opencv_transforms#8]
Not sure if I'm doing something wrong, but this gives an error:  "TypeError: 'int' object is not iterable"

```
transformations = transforms.Compose([
    transforms.Resize(255),
    # some other transforms
])
```

**Comment by Huud** [comment]
Ok, this tuple is breaking the function when using an int, and it is not in the original pytorch's transforms.py which just has "self.size = size":

https://github.com/jbohnslav/opencv_transforms/blob/620f1f4881173faa0fc8f884087ee20359754765/opencv_transforms/transforms.py#L126

**Comment by jbohnslav** [comment]
Thanks for your help! Fixed with [#29c6e35].(https://github.com/jbohnslav/opencv_transforms/commit/29c6e359330a1a522f0099563826d6ef812096a8)
